cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/objects-list-header.tsx

```diff
@@ -91,7 +113,12 @@ function ObjectListHeader(props: Props): JSX.Element {
                         <HideAllSwitcher {...props} />
                     </>
                 )}
-                <CollapseAllSwitcher {...props} />
+                <Row>
+                    {readonly && (
+                        <GTSwitcher {...props} />
+                    )}
```

Comment:
It is not clear how readonly should affect GTSwitcher
They have different semantic meaning
If you wanted to show GTSwitcher only in review mode, better to use workspace variable